improvedigitaladserverjsclient.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+var utils = require('./utils.js');
+function ImproveDigitalAdServerJSClient(endPoint) {
+  this.CONSTANTS = {
+    HTTP_REQUEST_TYPE: {
+      GET: 0,
+      POST: 1
+    },
+    HTTP_SECURITY: {
+      STANDARD: 0,
+      SECURE: 1
+    },
+    AD_SERVER_BASE_URL: 'ad.360yield.com',
+    END_POINT: endPoint || 'hb',
+    AD_SERVER_URL_PARAM: 'jsonp=',
+    CLIENT_VERSION: 'JS-4.3.3',
+    MAX_URL_LENGTH: 2083,
+    ERROR_CODES: {
+      BAD_HTTP_REQUEST_TYPE_PARAM: 1,
+      MISSING_PLACEMENT_PARAMS: 2,
+      LIB_VERSION_MISSING: 3
+    },
+    RETURN_OBJ_TYPE: {
+      DEFAULT: 0,
+      PREBID: 1
+    }
+  };
+
+  this.getErrorReturn = function (errorCode) {
+    return {
+      idMappings: {},
+      requests: {},
+      'errorCode': errorCode
+    };
+  };
+
+  this.createRequest = function (requestObject, requestParameters, extraRequestParameters) {
+    if (requestParameters.httpRequestType !== this.CONSTANTS.HTTP_REQUEST_TYPE.GET) {
+      return this.getErrorReturn(this.CONSTANTS.ERROR_CODES.BAD_HTTP_REQUEST_TYPE_PARAM);
+    }
+    if (!requestParameters.libVersion) {
```

Comment:
Same concern as the above comment.

Reply:
This will be removed in the next submission.